test/EFCore.PG.FunctionalTests/Query/SpatialQueryNpgsqlGeometryFixture.cs

```diff
@@ -0,0 +1,41 @@
+using GeoAPI.Geometries;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Query;
+using Microsoft.EntityFrameworkCore.TestUtilities;
+using Microsoft.Extensions.DependencyInjection;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Infrastructure;
+using Npgsql.EntityFrameworkCore.PostgreSQL.TestUtilities;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.Query
+{
+    public class SpatialQueryNpgsqlGeometryFixture : SpatialQueryRelationalFixture
+    {
+        protected override ITestStoreFactory TestStoreFactory
+            => NpgsqlTestStoreFactory.Instance;
+
+        // TODO: Use UseNetTopologySuite instead?
```

Comment:
Any reason not to?

Reply:
Removed the leftover comments, am just doing it the way the EF Core tests do it...